### PR TITLE
Fixed typo and added constant injection type to address issues

### PR DESCRIPTION
**Pull request by saskiad**
Small fixes to procedures file to address issues:
- craniotomy_coordinates type
- added constant injection type

### CODE REVIEW

File: schemas/procedures.json [hunk 1]
```diff
@@ -43,6 +43,12 @@
             "uniqueItems": true,
             "title": "Injections"
         },
+        "fiber_implants": {
+            "type": "array",
+            "items": {"$ref": "#/$defs/impant"},
```
Comment (jtyoung84) [inline schemas/procedures.json:48]: I think this is a typo (should be implant)

File: schemas/procedures.json [hunk 2]
```diff
@@ -76,6 +82,100 @@
...
+                "probes": {
+                    "type": "array",
+                    "items": {"$ref": "#/$defs/probe"},
+                    "uniqueItems": true,
+                    "title": "Probes"
+                }
+            },
+            "required": ["date", "experimenter_full_name","probes"]
+        },
+        "probes": {
```
Comment (jtyoung84) [inline schemas/procedures.json:107]: Given the ref on line 100, I think this should be "probe"

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
There were a couple of typos I highlighted

File: schemas/procedures.json (hunk 1 above)
Comment (saskiad) [reply schemas/procedures.json:48]: thanks - and sorry!

File: schemas/procedures.json (hunk 2 above)
Comment (saskiad) [reply schemas/procedures.json:107]: oh, this is actually something else. Too many different types of probes. Let me find a different name for one of these.

Comment (saskiad) [reply schemas/procedures.json:107]: ah, that's right. Fixing it now.

**Comment by saskiad** [comment]
I think I fixed them - sorry about that.

**Review by jtyoung84 — APPROVED** [review]
Looks good

**Pull request merged**